### PR TITLE
Added Villager class and test.

### PR DESCRIPTION
**Pull request by paulslss300**

### CODE REVIEW

File: src/test/java/VillagerTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
+ * This file contains sample JUnit test cases for Horse.java
```
Comment (Enidx) [inline src/test/java/VillagerTest.java]: Please change the documentation from Horse.java to Villager.java!

**Review by Enidx — CHANGES_REQUESTED** [review]

File: src/test/java/VillagerTest.java (hunk 1 above)
Comment (paulslss300) [reply src/test/java/VillagerTest.java]: edited.

File: src/main/java/Villager.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import java.util.List;
+
+public class Villager implements Tradable, Domesticatable {
+
```
Comment (Winnie1029) [inline src/main/java/Villager.java:6]: Maybe should add documentation for the Villager class

**Review by Winnie1029 — CHANGES_REQUESTED** [review]

File: src/main/java/Villager.java (hunk 2 above)
Comment (paulslss300) [reply src/main/java/Villager.java:6]: done

**Comment by paulslss300** [comment]
resolved issues found by Enid and Winnie.

**Pull request merged**